ts/packages/node/src/stores/XmlFile.ts

```diff
@@ -0,0 +1,41 @@
+import xml2js from 'xml2js';
+import { type Config } from '@configu/ts';
+import _ from 'lodash';
+import { FileConfigStore } from './File';
+
+export type XmlFileConfigStoreConfiguration = { path: string };
+
+export class XmlFileConfigStore extends FileConfigStore {
+  constructor({ path }: XmlFileConfigStoreConfiguration) {
+    const initialFileState = `
+<?xml version="1.1" encoding="UTF-8"?>
+<root/>
+    `;
+    super('xml-file', { path, initialFileState });
+  }
```

Comment:
Let's allow users of the store to customize the builder according to their requirements.

```suggestion
export type XmlFileConfigStoreConfiguration = { path: string; builderOptions?: xml2js.BuilderOptions };

export class XmlFileConfigStore extends FileConfigStore {
  private builder: xml2js.Builder;
  constructor({ path, builderOptions }: XmlFileConfigStoreConfiguration) {
    const builder = new xml2js.Builder(builderOptions);
    const initialFileState = builder.buildObject({ root: { config: [] } });
    super('xml-file', { path, initialFileState });
    this.builder = builder;
  }
```